package.json

```diff
@@ -99,12 +99,13 @@
   },
   "devDependencies": {
     "@babel/core": "~7.16.7",
-    "@storybook/addon-actions": "^6.4.13",
-    "@storybook/addon-essentials": "^6.4.13",
-    "@storybook/addon-links": "^6.4.13",
-    "@storybook/builder-webpack5": "^6.4.13",
-    "@storybook/manager-webpack5": "^6.4.13",
-    "@storybook/react": "^6.4.13",
+    "@storybook/addon-actions": "^6.4.18",
+    "@storybook/addon-docs": "^6.4.18",
```

Comment:
On the documentation page for the `docs` addon, it says:
> we recommend that you install [@storybook/addon-essentials](https://www.npmjs.com/package/@storybook/addon-essentials), to include this and other great features into your project.

([source](https://storybook.js.org/docs/react/writing-docs/introduction))... 

The `docs` addon was already in our lock file so it seems like it is a dependency of some other dependency... are you sure we need to explicitly list it as a dependency?

Reply:
good point👍

Reply:
I was 100% sure that the build didn't work last week because this addon was missing. I just checked it locally, it works without the extra dependency. So weird 😖 

Will remove it again, thanks for the double check.